README.md

```diff
@@ -45,6 +45,75 @@ Once a `Client` is obtained queries can be made using the `query()` function. Th
 
 A custom timeout can be set by passing the `timeout` parameter in the `query` function call. If no timeout is specified, it will default to a 10 min timeout then cancel the query, and a TimeoutError exception will be raised.
 
+### HTTP API
+#### Prices
+Get the latest price for a token pair.
+```python
+>>> data = client.prices.get_latest("BTC-USDC")
```

Comment:
```suggestion
>>> client.prices.get_latest("BTC-USDC")
```

Shouldn't this not assign it to a variable? Otherwise the output wouldn't print like you've shown.

Reply:
Similar comment for below